src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum;
+
+  if (days >= 1 && days < 3) {
+    sum = 40 * days;
```

Comment:
It's better to create variables for every value (such as 40) to improve readability and maintenance. 
For example `const dailyCost = 40;`